utils/coreutils/utils.go

```diff
@@ -287,6 +287,14 @@ func GetJfrogPluginsDir() (string, error) {
 	return filepath.Join(homeDir, JfrogPluginsDirName), nil
 }
 
+func GetJfrogLocksDir() (string, error) {
```

Comment:
I suggest adding a new ```GetJfrogConfigLocksDir``` function (which uses the ```GetJfrogLocksDir``` function) right after this function. This will allow us to remove the "join" from the code which creates the config lock, and also remove the ```config.ConfigLockDirName``` const.